Microsoft.DotNet.Try.Markdown/CodeFenceAnnotationsParser.cs

```diff
@@ -82,37 +82,49 @@ public class CodeFenceAnnotationsParser
 
         private Parser CreateOptionsParser(Action<Command> configureCsharpCommand = null)
         {
-            var packageOption = new Option(packageOptionName,
+            var packageOption = new Option(PackageOptionName,
                                            argument: new Argument<string>());
 
-            if (defaultAnnotations?.Package is string defaultPackage)
+            if (_defaultAnnotations?.Package is string defaultPackage)
             {
                 packageOption.Argument.SetDefaultValue(defaultPackage);
             }
 
-            var packageVersionOption = new Option(packageVersionOptionName,
+            var packageVersionOption = new Option(PackageVersionOptionName,
                                                   argument: new Argument<string>());
 
-            if (defaultAnnotations?.PackageVersion is string defaultPackageVersion)
+            if (_defaultAnnotations?.PackageVersion is string defaultPackageVersion)
             {
                 packageVersionOption.Argument.SetDefaultValue(defaultPackageVersion);
             }
 
+            var languageCommands = new[]
+            {
+                CreateCsharpCommand(configureCsharpCommand, packageOption, packageVersionOption),
+                CreateFsharpCommand(configureCsharpCommand, packageOption, packageVersionOption)
```

Comment:
This shouldn't pass `configureCsharpCommand` or else the parameter should be renamed if the configurations are intended to apply to all subcommands.